src/features/components/templates/navigation/template/index.tsx

```diff
@@ -58,60 +56,56 @@ const NavTemplate = ({
+                    <NavProvider
+                        is_menu_open={is_menu_open}
+                        onCloseMenu={onMenuToggleClick}
+                        items={items}
                     >
-                        <NavProvider
-                            is_menu_open={is_menu_open}
-                            onCloseMenu={onMenuToggleClick}
-                            items={items}
+                        <Flex.Box
+                            justify="center"
+                            align="center"
+                            grow={has_centered_logo && items.length === 0 ? '1' : undefined}
                         >
-                            <Flex.Box
-                                justify="center"
-                                align="center"
-                                grow={has_centered_logo && items.length === 0 ? '1' : undefined}
-                            >
-                                {has_items ? (
-                                    <Flex.Item visible="phone-and-tablet" innerRef={nav_toggle_ref}>
-                                        <MobileMenuToggle
-                                            is_open={is_menu_open}
-                                            onClick={onMenuToggleClick}
-                                        />
-                                    </Flex.Item>
-                                ) : null}
-                                {renderLogo()}
-                            </Flex.Box>
-
                             {has_items ? (
-                                <>
-                                    {is_mobile_or_tablet ? (
-                                        <span ref={nav_wrapper_ref}>
-                                            <MobileMenu has_top_nav={has_top_nav} />
-                                        </span>
-                                    ) : (
-                                        <DesktopMenu has_centered_items={has_centered_items} />
-                                    )}
-                                </>
+                                <Flex.Item visible="phone-and-tablet" innerRef={nav_toggle_ref}>
+                                    <MobileMenuToggle
+                                        is_open={is_menu_open}
+                                        onClick={onMenuToggleClick}
+                                    />
+                                </Flex.Item>
                             ) : null}
-                            {children}
-                        </NavProvider>
-                    </Flex.Box>
-                    {render_bottom_nav?.()}
-                </Container.Fixed>
-            ) : (
-                <Flex.Box className={styles.no_nav} />
-            )}
+                            {renderLogo()}
+                        </Flex.Box>
+
+                        {has_items ? (
+                            <>
+                                {is_mobile_or_tablet ? (
```

Comment:
hi @konstantinosG-deriv  we used is_mobile_or_tablet and faced lots of issue in rendering , instead of that we can use   <div className="visible-phone-and-tablet"/>

Reply:
please keep this PR clean, we can address the other issue in another PR, thanks.